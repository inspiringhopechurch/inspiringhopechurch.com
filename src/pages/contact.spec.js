import React from "react";
import { shallow } from "enzyme";
import Contact from "./contact";

describe("Contact", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Contact />);
    expect(wrapper).toMatchSnapshot();
  });
});
