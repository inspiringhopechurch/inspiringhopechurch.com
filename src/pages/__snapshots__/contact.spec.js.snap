// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should render without crashing 1`] = `
<Fragment>
   
  <SEO
    article={false}
    banner={null}
    desc={null}
    pathname={null}
    title="Contact Us"
  />
  <section
    className="contact-page fade-in hero is-halfheight"
  >
    <div
      className="hero-body"
    >
      <div
        className="container has-text-centered"
      >
        <FancyHeading
          heading="Contact Us"
        />
      </div>
    </div>
  </section>
  <section
    className="contact-page contact-content section is-halfheight"
  >
    <div
      className="columns content is-centered"
    >
      <div
        className="column is-one-third-tablet mb-4"
      >
        <FancyHeading
          className="has-text-centered"
          heading="Have Questions?"
        />
        <p>
          Do you have questions about Inspiring Hope Church? Would like to join one of our groups? 
          <br />
          Please feel free to call, reach us on facebook, or use the form on this page to get in touch!
        </p>
        <p>
          <span
            className="icon"
          >
            <FontAwesomeIcon
              beat={false}
              beatFade={false}
              border={false}
              bounce={false}
              className=""
              fade={false}
              fixedWidth={false}
              flip={null}
              icon={
                Array [
                  "fas",
                  "map-marker-alt",
                ]
              }
              inverse={false}
              listItem={false}
              mask={null}
              maskId={null}
              pull={null}
              pulse={false}
              rotation={null}
              shake={false}
              size="lg"
              spin={false}
              swapOpacity={false}
              symbol={false}
              title=""
              titleId={null}
              transform={null}
            />
          </span>
           Based in Hamilton, Ohio.
        </p>
        <p>
          <a
            href="tel:19189318591"
            title="Get in touch!"
          >
            <span
              className="icon"
            >
              <FontAwesomeIcon
                beat={false}
                beatFade={false}
                border={false}
                bounce={false}
                className=""
                fade={false}
                fixedWidth={false}
                flip="horizontal"
                icon={
                  Array [
                    "fas",
                    "phone",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                maskId={null}
                pull={null}
                pulse={false}
                rotation={null}
                shake={false}
                size="lg"
                spin={false}
                swapOpacity={false}
                symbol={false}
                title=""
                titleId={null}
                transform={null}
              />
            </span>
             1 (918) 931-8591
          </a>
        </p>
        <p>
          <a
            href="https://www.facebook.com/inspiringhopechurch/"
            title="Link to Facebook page"
          >
            <span
              className="icon"
            >
              <FontAwesomeIcon
                beat={false}
                beatFade={false}
                border={false}
                bounce={false}
                className=""
                fade={false}
                fixedWidth={false}
                flip={null}
                icon={
                  Array [
                    "fab",
                    "facebook-f",
                  ]
                }
                inverse={false}
                listItem={false}
                mask={null}
                maskId={null}
                pull={null}
                pulse={false}
                rotation={null}
                shake={false}
                size="lg"
                spin={false}
                swapOpacity={false}
                symbol={false}
                title=""
                titleId={null}
                transform={null}
              />
            </span>
             Inspiring Hope Facebook page
          </a>
        </p>
      </div>
      <div
        className="column is-two-thirds-tablet is-two-fifths-widescreen"
      >
        <ContactForm
          bodyValidationMsg="Tell us how we can help you."
          emailValidationMsg="Please enter your email"
          formTitle="Get in Touch"
          nameValidationMsg="Please enter your full name"
          subjectValidationMsg="Tell us why you're reaching out."
          submitButtonTitle="Send Message"
        />
      </div>
    </div>
  </section>
</Fragment>
`;
